indico/util/date_time_test.py

```diff
@@ -70,8 +71,18 @@ def test_iterdays(from_, to, skip_weekends, day_whitelist, day_blacklist, expect
     assert len(list(iterdays(from_, to, skip_weekends, day_whitelist, day_blacklist))) == expected
 
 
+@pytest.mark.parametrize(('skeleton', 'format', 'expected'), (
+    ('MMM', 'M', 'M'),  # Cannot expand numeric fields to alphabetic ones
+    ('ss', 's', 's'),  # Hours, minutes and seconds should never be expanded
+    ('MMdd', 'Md', 'MMdd'),  # Expand numeric fields
+    ('MMMMdddd', 'MMMddd', 'MMMMdddd')  # Expand alphabetic fields
+))
+def test__adjust_skeleton_skeleton(skeleton, format, expected):
```

Comment:
```suggestion
def test_adjust_skeleton_skeleton(skeleton, format, expected):
```

not a fan of that double underscore :p